src/main/java/Duke.java

```diff
@@ -1,10 +1,331 @@
+                String line = fileReader.nextLine();
+                char taskType = line.charAt(4);
+                char doneStatus = line.charAt(7);
+                String subLine = line.substring(10);
+                switch (taskType) {
+                    case 'T':
+                        Todo t = new Todo(subLine);
+                        if (doneStatus == 'X') {
+                            t.setDone();
+                        }
+                        allTasks.add(t);
+                        break;
+                    case 'D':
+                        String[] deadlineParts = subLine.split(" \\(by: ");
+                        Deadline d = new Deadline(deadlineParts[0], deadlineParts[1].substring(0, deadlineParts[1].length() -1));
+                        if (doneStatus == 'X') {
+                            d.setDone();
+                        }
+                        allTasks.add(d);
+                        break;
+                    case 'E':
+                        String[] eventParts = subLine.split(" \\(at: ");
+                        Event e = new Event(eventParts[0], eventParts[1].substring(0, eventParts[1].length()-1));
+                        if (doneStatus == 'X') {
+                            e.setDone();
+                        }
+                        allTasks.add(e);
+                        break;
+                    default:
+                        break;
+                }
+            }
+            fileReader.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("An error occurred.");
+            e.printStackTrace();
+        }
+    }
+
+
+    public static void main(String[] args) throws FileNotFoundException {
+
+        String action = null;
+        Scanner input = new Scanner(System.in);
+        int taskNum = 0;
+        String filePath = "/Users/yanyixue/Desktop/year3/sem2/CS2113/ip/tasks.txt";
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("----------------------------------------------------");
+        System.out.println(logo);
+        System.out.println("Aloha! I'm Duke");
+        System.out.println("What would you like to do today?");
+        System.out.println("Enter 'Bye' when done.");
+        System.out.println("----------------------------------------------------");
+
+        // obtain string input
+        if(!input.hasNext() || (action = input.nextLine()).isEmpty()) System.exit(0);
+
+        File f = new File(filePath);
+        if(f.exists() && !f.isDirectory()) {
+            ReadFile(f);
+        }
+
+        while (!action.equalsIgnoreCase("Bye")){
+            String actionArr[] = action.split(" ", 2);
+            switch(actionArr[0].toLowerCase()){
```

Comment:
Consider spacing out the brackets more evenly. For example,
`while (!action.equalsIgnoreCase("Bye")) {`